### ISSUE TITLE
L3 edge disappears under specific VLAN configuration

### ISSUE THREAD

**Issue by saparikh** [batfish/batfish#6994]
Snapshot:
[basic_1.zip](https://github.com/batfish/batfish/files/6460816/basic_1.zip)

If you remove the `members vlan11` command from under `xe-0/0/0`  and `members vlan21` from under `xe-0/0/1` in `sw-wan-1.cfg` you will get these L3 edges:

```
bf.q.layer3Edges().answer().frame()
Out[31]: 
                Interface           IPs        Remote_Interface    Remote_IPs
0  rtr-isp-1[xe-0/0/1.20]  ['10.0.0.2']  rtr-1[xe-0/0/0.10]      ['10.0.0.1']
1  rtr-1[xe-0/0/0.10]      ['10.0.0.1']  rtr-isp-1[xe-0/0/1.20]  ['10.0.0.2']
```

if you add either of those `vlan` statements back, the edges disappear.

**Comment by corinaminer** [comment]
Fixed in #6992.

**Comment by corinaminer** [comment]
cc: @sharonsa -- thanks for reporting :)
